data/goal_src/jak1/engine/common-obs/nav-enemy.gc

```diff
@@ -1289,6 +1289,12 @@ nav-enemy-default-event-handler
     (none)
     )
   :code (behavior ()
+    ;; no killing in pacifist, except for finalboss lurkers
+    (if (and (-> *challenges-settings* challenges-pacifist?)
+             (not (= (-> self type symbol) 'green-eco-lurker))
+             )
+      (reset-all)
+      )
```

Comment:
main pacifist check - other enemies that aren't children of nav-enemy have similar code in their respective XYZ-die state

the only exceptions that I (consciously) made to this are:
- TNT lurker at end of mountain pass
- finalboss lurkers (thats green-eco-lurker)
- finalboss